### DISCUSSION TITLE
Duda y busco equipo

### DISCUSSION THREAD

**Discussion by denissegutierrez** [Launch-X-Latam/MisionFrontEnd#566] [Team Posts]
Buenas tardes compañeros, si alguien me pudiera indicar como hacer para acceder a BACKEND en teams se lo agradeceria un monton, tambien estoy buscando un EQUIPO que este comprometido para el hackaton

**Comment by luisjacobpy** [comment]
Hola, yo tambien busco equipo

**Comment by CarlosTochi** [comment]
Hola yo tambien busco equipo 

Te dejo los pasos para que puedas ingresar a la mision backend https://teams.microsoft.com/l/message/19:e7c1d5fb5f2940ae802e60b463a59d0b@thread.tacv2/1665461397985?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=8c234d62-8e08-415a-8242-509fa69c5c40&parentMessageId=1665461397985&teamName=Launch%20X%3A%20Misi%C3%B3n%20FrontEnd&channelName=Avisos%20GitHub&createdTime=1665461397985&allowXTenantAccess=false

Este es mi correo de innovaccion lx02273@innovaccion.mx

**Comment by luisjacobpy** [comment]
Les dejo mi email: lx04550@innovaccion.mx

**Comment by Ayuya2001** [comment]
Hola, yo estoy sin equipo, ahora en estos momentos me estoy poniendo al día
en los 2 cursos en los que estoy, quiero darle con todo al hackaton
Les dejo mis correos ***@***.***
***@***.***
***@***.***

**Comment by IngDavidHoyosGil** [comment]
Hola! acabo de crear un equipo en teams con los dos compañeros que dejaron su email, los demas favor de dejar su email para el hackaton.

Muchas gracias!

**Comment by Cposadas05** [comment]
Hola amigo, me puedes agregar por favor ***@***.***

Enviado desde mi Samsung Mobile de Claro
Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by H3nt4ur0** [comment]
Buenas Tardes, todavía no tengo ningún equipo mi correo es lx09980@innovaccion.mx

**Comment by Ayuya2001** [comment]
Ya no te quedan espacios? Por si quedarán este es mi correo
***@***.***

**Comment by IngDavidHoyosGil** [comment]
Nos queda un cupo, pero por favor alguien escribir desde el discussions de Github, que desde el correo no nos permite ver la direccion del email

**Comment by iseddev** [comment]
Este es mi correo: lx01741@innovaccion.mx

**Comment by xramons** [comment]
https://teams.microsoft.com/l/team/19%3a59Tb8L_0PlM5uxQtC8AjVfcF-wvYCKBxsS1JRWZLnyo1%40thread.tacv2/conversations?groupId=e6804a94-ce40-41ca-a5c2-b287fc50036b&tenantId=4ae54b05-b77e-4224-aef1-8661422e0816

**Comment by denissegutierrez** [comment]
lx00689@innovaccion.mx
este es mi correo

**Comment by denissegutierrez** [comment]
lx00689@innovaccion.mx
 este es mi correo y muchas gracias por la liga :)
